### PR TITLE
Generate 24-word mnemonics

### PR DESCRIPTION
**Pull request by lukaw3d**
Fixes https://github.com/oasisprotocol/oasis-wallet-ext/issues/199
Mitigates https://github.com/oasisprotocol/oasis-wallet-ext/issues/149

### CODE REVIEW

File: src/popup/pages/ShowMnemonic/index.scss [hunk 1]
```diff
@@ -26,7 +26,7 @@
 
 .mne-item {
   padding: 8px 15px;
-  margin: 5px 15px 5px 0;
+  margin: 5px 10px 5px 0;
```
Comment (pro-wh) [inline src/popup/pages/ShowMnemonic/index.scss]: move these to the styling commit? or is this something else

**Review by pro-wh — APPROVED** [review]
you can still restore from 12-word mnemonic right?

**Comment by lukaw3d** [comment]
Yep, can still restore with 12-word mnemonics; and wording in restore is

> "inputSeed": "Please enter 12 or 24 seed words in order, with each word separated by a space.",

File: src/popup/pages/ShowMnemonic/index.scss (hunk 1 above)
Comment (lukaw3d) [reply src/popup/pages/ShowMnemonic/index.scss]: Split into two commits, and renamed a bit:
```diff
- Generate 24-word mnemonics
+ Generate 24-word mnemonics
+ Display smaller mnemonic words to fit average 24-word mnemonics

 Simplify .mne-item styling
 
- Make popup wider so 24-word mnemonics don't overflow
+ Make popup wider to fit widest 24-word mnemonics
```

**Pull request merged**